docs/troubleshooting/README.md

```diff
@@ -1 +1,114 @@
-TBA.
\ No newline at end of file
+# Troubleshooting guide
```

Comment:
Hi Arvind! Let's talk more about this in our sync today; thanks for the doc work 🙏